static/docs/command-reference/import-url.md

```diff
@@ -31,8 +31,10 @@ location into the workspace, such that it will be automatically updated (by
 
 The `dvc import-url` command helps the user create such an external data
 dependency. The `url` argument specifies the external location of the data to be
-imported, while `out` can be used to specify the (path and) file name desired
-for the imported data file or directory in the workspace.
+imported, while `out` can be used to specify the directory and/or file name
+desired for the imported data file or directory in the workspace. If an existing
+directory is specified (without a file name), then the output will be placed
+inside of it.
```

Comment:
Let's make this exactly the same as whatever ends up being in static/docs/command-reference/get-url.md? Both should be wrapped to 80 chars.